_docs/021-getting-started-gke.md

```diff
@@ -107,7 +106,8 @@ gcloud container clusters create $CLUSTER_NAME \
   --enable-autoscaling --min-nodes=1 --max-nodes=3 \
   --enable-autorepair \
   --scopes=service-control,service-management,compute-rw,storage-ro,cloud-platform,logging-write,monitoring-write,pubsub,datastore \
-  --num-nodes=3
+  --num-nodes=3 \
+  --region=$(gcloud config get-value compute/region)
```

Comment:
this could be `--region=$GCP_REGION` if we avoid changing the defaults

Reply:
One thing to consider here is that the initial number of nodes seems to be 9 if I specify region while it is 3 if I specify zone, so should we set a GCP_ZONE variable instead and use `--zone=$GCP_ZONE` instead?

```
NAME      LOCATION  MASTER_VERSION  MASTER_IP     MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
riff-gke  us-east1  1.12.6-gke.7    35.231.35.41  n1-standard-2  1.12.6-gke.7  9          RUNNING
```

```
NAME      LOCATION       MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
riff-new  us-central1-a  1.12.5-gke.5    35.188.219.17  n1-standard-2  1.12.5-gke.5  3          RUNNING
```

Reply:
I wonder why master and node versions are different? `1.12.6-gke.7` vs `1.12.5-gke.5`, does it vary by region?

Reply:
i just got this 3 node cluster
```
$ gcloud container clusters list
NAME            LOCATION      MASTER_VERSION  MASTER_IP     MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
jldec-riff-030  europe-west1  1.12.5-gke.5    35.233.77.14  n1-standard-2  1.12.5-gke.5  3          RUNNING
```
using
```
$ gcloud container clusters create $CLUSTER_NAME \
>   --cluster-version=latest \
>   --machine-type=n1-standard-2 \
>   --enable-autoscaling --min-nodes=1 --max-nodes=3 \
>   --enable-autorepair \
>   --scopes=service-control,service-management,compute-rw,storage-ro,cloud-platform,logging-write,monitoring-write,pubsub,datastore \
>   --num-nodes=3 \
>   --region=$GCP_REGION
```
(where GCP_REGION is europe-west)

Reply:
in us-east1 i get 9 nodes just like you did
and doing a build in the europe-west1 cluster scaled it to 4
i'm confused. 
```
NAME                     LOCATION      MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
jldec-riff-030           europe-west1  1.12.5-gke.5    35.233.77.14   n1-standard-2  1.12.5-gke.5  4          RUNNING
jldec-riff-030-us-east1  us-east1      1.12.6-gke.7    35.196.147.71  n1-standard-2  1.12.6-gke.7  9          RUNNING
```

Reply:
looking in the console to figure out what's going on ... 
`--min-nodes` and `--max-nodes` applies per zone, not per cluster

you were right @trisberg  - according to https://cloud.google.com/sdk/gcloud/reference/container/clusters/create

> If not specified, all nodes will be in the cluster's primary zone (for zonal clusters) or spread across three randomly chosen zones within the cluster's region (for regional clusters).

Reply:
It does scale down after a few minutes and now I have 3 in us-east1, so let's go with region for now

Reply:
Actually i prefer single zone since this is intended to be a minimal test cluster.